### ISSUE TITLE
feature request: mamba

### ISSUE THREAD

**Issue by nick-youngblut** [Bioconductor/basilisk#42]
mamba is substantially faster than conda, and moreover, a micromamba install is smaller and faster to install versus miniconda.

**Comment by LTLA** [comment]
Check out the discussion at Bioconductor/basilisk.utils#11; from what I understand, the `miniforge` setup uses `mamba` under the hood. The idea is to have this become the default in the next release, assuming that the inevitable struggles on Windows can be resolved.

**Comment by nick-youngblut** [comment]
> The idea is to have this become the default in the next release

That's great! Thanks for the info.

> inevitable struggles on Windows can be resolved

Does anyone actually do bioinformatics on Windows machines? ...especially when running Linux VMs on a cloud service is so easy and cheap?

**Comment by LTLA** [comment]
> Does anyone actually do bioinformatics on Windows machines? ...especially when running Linux VMs on a cloud service is so easy and cheap? There's also WSL, so why use Windows?

Apparently 50% of BioC package downloads are for the Windows binaries, as of a few years ago - @vjcitn could give more recent statistics. So, not a negligible proportion of users. I'd guess they're probably bench scientists just trying to get something done with whatever machine they were given; it would be a pretty high barrier to entering bioinformatics if they needed a cloud account (I don't even have one) or set up WSL (I haven't) to get started.

**Comment by vjcitn** [comment]
@nick-youngblut Note that you can get faster installation on non-windows machines now if you set the environment variable 
```
BASILISK_MINICONDA_VERSION=py311_23.11.0-2
```
see section 3.2 of the basilisk vignette.  That version of miniconda actually uses mamba resolution IIUC.

**Comment by vjcitn** [comment]
To take advantage of this you might have to remove content in the cache  at basilisk.utils::getExternalDir()

**Comment by nick-youngblut** [comment]
Thanks @vjcitn for the advice!

**Comment by nick-youngblut** [comment]
> To take advantage of this you might have to remove content in the cache at basilisk.utils::getExternalDir()

Yeah, both setting `BASILISK_MINICONDA_VERSION` and checking `basilisk.utils::getExternalDir()` is definitely a bit of a burden for users.

**Comment by LTLA** [comment]
Miniforge 24.3.0-0 is now used by default in BioC-devel (as of 1.17.1), which should be using libmamba under the hood. This should improve environment setup speed.
